tests/phpunit/tests/post/updatePostCache.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Test `wp_update_cache()`.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Test class for `wp_update_cache()`.
+ *
+ * @ticket 50567
+ *
```

Comment:
```suggestion
```

Further tests may be added that don't relate to this ticket.

Each of the tests include this in their docblock so no practical affect for now.